build.gradle

```diff
@@ -31,6 +28,7 @@ dependencies {
   testCompile 'io.dropwizard:dropwizard-testing:1.3.5'
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:2.19.0'
+  testCompile 'com.h2database:h2:1.4.197'
```

Comment:
Did we want to test alternative DB types that we currently don't support? The DB [schema](https://github.com/MarquezProject/marquez/blob/master/src/main/resources/db/migration/V1__create_tables.sql) is currently defined using postgres. We also config [postgres](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/MarquezApplication.java#L56) at runtime and don't allow for other DB types (at least currently).

Reply:
Ahh, overlooked `testCompile`. What are you're thoughts on using [testcontainers](https://www.testcontainers.org/) instead of H2?

Reply:
I was hoping that h2 would be compatible enough. I did not know about testcontainers, but as long as the setup is integrated in the test (with a  Rule) then it looks great. We just need to verify it what dependencies are (docker installed etc). The fewer the better.

Reply:
There is a dependency on docker, which does add _one more thing_ to get up and running. That said, for integration tests, that'd be fine and expected.